src/components/tree-detail/tree-water-needs/contact-dialog.tsx

```diff
@@ -0,0 +1,177 @@
+/* eslint-disable max-lines */
+import React, { useCallback, useState } from "react";
+import { useI18nStore } from "../../../i18n/i18n-store";
+import { AlertDialog } from "../../alert-dialog/alert-dialog";
+import { PrimaryButton } from "../../buttons/primary";
+import { TertiaryButton } from "../../buttons/tertiary";
+import { CheckIcon } from "../../icons/check-icon";
+import { CloseIcon } from "../../icons/close-icon";
+import Markdown from "react-markdown";
+import { supabaseClient } from "../../../auth/supabase-client";
+
+const closeContactDialog = () => {
+	(document.getElementById("contact-dialog") as HTMLDialogElement).close();
+};
+
+interface ContactDialogProps {
+	contactUsername: string;
+}
+export const ContactDialog: React.FC<ContactDialogProps> = ({
+	contactUsername,
+}) => {
+	const [message, setMessage] = useState("");
+	const [isContactRequestLoading, setIsContactRequestLoading] = useState(false);
+	const [alreadySentContact, setAlreadySentContact] = useState(false);
+	const [error, setError] = useState("");
```

Comment:
same point as from our tests - are you using "error" anywhere?

Reply:
fixed in latest commit